src/app/api/tools/route.ts

```
@@ -0,0 +1,39 @@
+    )
+  }
+
+  // Sort tools
+  switch (sortBy) {
+    case 'stars':
+      filteredTools.sort((a, b) => b.githubStars - a.githubStars)
+      break
+    case 'lastUpdate':
+      filteredTools.sort((a, b) => new Date(b.lastUpdate).getTime() - new Date(a.lastUpdate).getTime())
+      break
+    case 'relevance':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)